scripts/makeLegacyPlots_postFit.py

```diff
@@ -0,0 +1,935 @@
+
+    hothersList = [hEWK, hsingleT, hTW, hVV, hTTX, hVVV, hWJets]
+    hSingleHlist = [hZH, hWH, hTTH, hggH, hqqH]
+    hSingleH = makeSum("singleH",hSingleHlist)
+    hothers = makeSum("other",hothersList)
+
+    hBkgList     = [hothers , hSingleH, hQCD, hTT, hDY]                   # list for stack
+    hBkgNameList = ["Others", "Single H", "QCD", "t#bar{t}", "Drell-Yan"] # list for legend
+
+    gData = getHisto("data", hDatas , doOverflow) # NB - data is stored as a TGRaphAsymErrors inside a ROOFit object
+    gDataNonScaled = gData.Clone("gDataNonScaled")
+
+
+    ######################### SET COLORS ####################
+    sigColors = {}
+    sigColors["ggHH_kl_1_kt_1_hbbhtt"] = kBlack
+    sigColors["qqHH_CV_1_C2V_1_kl_1_hbbhtt"] = kCyan
+
+    col = TColor()
+    bkgColors = {}
+    bkgColors["DY"]    = col.GetColor("#44BA68")
+    bkgColors["TT"]    = col.GetColor("#F4B642")
+    bkgColors["singleH"]     = col.GetColor("#41B4DB")
+    bkgColors["other"] = col.GetColor("#ED635E")
+    bkgColors["QCD"]     = col.GetColor("#F29563")
+
+    bkgLineColors = {}
+    bkgLineColors["DY"]    = col.GetColor("#389956")
+    bkgLineColors["TT"]    = col.GetColor("#dea63c")
+    bkgLineColors["singleH"]     = col.GetColor("#3ca4c8")
+    bkgLineColors["other"] = col.GetColor("#d85a56")
+    bkgLineColors["QCD"] = col.GetColor("#DC885A")
+
+    # apply sig color if available
+    for key in hSigs:
+        hSigs[key].SetLineWidth(2)
+        if doOverflow: hSigs[key] = addOverFlow(hSigs[key])
+        if key in sigColors:
+            thecolor = int(sigColors[key])
+            hSigs[key].SetLineColor(thecolor)
+
+
+    # apply bkg color if available
+    for h in hBkgList:
+            histoname =h.GetName()
+            for key,value in bkgColors.items():
+                if key in histoname:    
+                        thecolor = int(bkgColors[key])
+                        h.SetFillColor(thecolor)
+                        h.SetFillStyle(1001)
+
+    # apply bkg lines colors if available
+    for h in hBkgList:
+        histoname = h.GetName()
+        for key,value in bkgLineColors.items():
+            if key in histoname:
+                thecolor = int(bkgLineColors[key])
+                h.SetLineColor(thecolor)
+                h.SetLineWidth(1)
+
+
+    #################### REMOVE NEGARIVE BINS #######################
+    for h in hBkgList: print "Integral ", h.GetName(), " : ", h.Integral(), " - ", h.Integral(-1,-1)
+    for n in hDatas: print "Integral ", hDatas[n].GetName(), " : ", hDatas[n].Integral(), " - ", hDatas[n].Integral(-1,-1)
+    for i, name in enumerate (sigNameList): print "Integral ", hSigs[sigList[i]].GetName(), " : ", hSigs[sigList[i]].Integral(), " - ", hSigs[sigList[i]].Integral(-1,-1)
+    
+    # Store yields in a txt file for reference
+    with open('./LegacyPlots/Legacy' + args.year + '/' + args.channel + '_' + args.tag + '/' + args.sel+'/yields_'+args.var+'_'+args.sel+'_'+args.reg+'.txt','w') as yields_file:
```

Comment:
maybe add "_prefit/" or "_postfit" to the directory name? 
Just after `args.channel + '_' + args.tag` seems a good spot.

Reply:
please also add `args.prepost` in this line (and the next one!)